Makefile

```diff
@@ -54,7 +54,7 @@ windows:
 
 deploy:
 	fly scale count 2 --region lax --yes
-	docker build -t paisa . --file Dockerfile.demo
+	docker build . -t paisa:demo --file Dockerfile.demo
```

Comment:
if the tag is changed here, it should be changed below as well, but I would suggest not to change anything related to demo.

Reply:
Done. :)